src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -131,9 +131,9 @@ public HikariPool(final HikariConfig config)
          this.houseKeepingExecutorService = config.getScheduledExecutorService();
       }
 
-      this.houseKeepingExecutorService.scheduleWithFixedDelay(new HouseKeeper(), 0L, HOUSEKEEPING_PERIOD_MS, MILLISECONDS);
+      this.leakTask = new ProxyLeakTask(config.getLeakDetectionThreshold());
 
-      this.leakTask = new ProxyLeakTask(config.getLeakDetectionThreshold(), houseKeepingExecutorService);
+      this.houseKeepingExecutorService.scheduleWithFixedDelay(new HouseKeeper(), 0L, HOUSEKEEPING_PERIOD_MS, MILLISECONDS);
```

Comment:
What is the reason for switching these?  Functionally they are the same, but it reads more cleanly if all of the ``this.houseKeepExecutorService`` initialization is grouped together (next to lines 124-132).